packages/labs/src/components/select/select.tsx

```diff
@@ -144,10 +150,11 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
     public render() {
         // omit props specific to this component, spread the rest.
         const {
+            closeOnSelect,
             filterable,
             initialContent,
-            itemRenderer,
             inputProps,
+            itemRenderer,
```

Comment:
👀

Reply:
this is legit, correctly alphabetized now 👍